locations/spiders/adopt.py

```
@@ -0,0 +1,27 @@
+
+from locations.structured_data_spider import StructuredDataSpider
+from locations.categories import Categories, apply_category
+
+
+
+class AdoptSpider(SitemapSpider, StructuredDataSpider):
+    name = "adopt"
+    item_attributes = {
+        "brand": "Adopt'",
+        "brand_wikidata": "Q104649177",
+    }
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)